provider/resource_opensearch_dashboard_object.go

```diff
@@ -43,23 +43,23 @@ func resourceOpensearchKibanaObject() *schema.Resource {
 					}
 
 					for _, o := range body {
-						kibanaObject, ok := o.(map[string]interface{})
+						DashboardObject, ok := o.(map[string]interface{})
```

Comment:
this shouldn't be capitalized: `dashboardObject`. Please fix all occurrences

Reply:
Thanks fixed this.